getting-started-gradle/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id 'java'
+    id 'io.quarkus'
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+}
+
+dependencies {
+    implementation group: 'io.quarkus', name: 'quarkus-resteasy', version: quarkusVersion
+
+    testCompile group: 'io.quarkus', name: 'quarkus-junit5', version: quarkusVersion
+    testCompile group: 'io.rest-assured', name: 'rest-assured', version: '3.3.0'
```

Comment:
How did you create the project? Because this is a very old version of REST Assured and from what I can see, the project doesn't look like a project our tools generate (no enforced BOM typically)?